docs/book/src/usage/setting-lint-levels.md

```diff
@@ -3,52 +3,20 @@ Once your crate is configured to run some lints, it quickly becomes important to
 code. Marker provides the ability to use normal lint control attributes like `#[allow(...)]` `#[deny(...)]` and others
 to control the behavior of marker lints.
 
-## On nightly
-Marker uses the `marker::` tool prefix for lints. In order for rustc to recognize this prefix, it requires marker to be
-registered via the [`#[register_tool()]` feature](https://github.com/rust-lang/rust/issues/66079).
+Marker uses the `marker::` tool prefix for lints. You can then use all of the normal lint attributes you might usually use, on marker provided lints, like `#[allow(marker::my_lint)]`, but you have to put them under `#[cfg_attr(marker, ...)]`. This is because normally `rustc` doesn't know anything about `marker` and it will complain about an unknown lint if yo u run simple `cargo check/build`.
```

Comment:
```suggestion
Marker uses the `marker::` tool prefix for lints. You can then use all of the normal lint attributes you might usually use, on marker provided lints, like `#[allow(marker::my_lint)]`, but you have to put them under `#[cfg_attr(marker, ...)]`. This is because normally `rustc` doesn't know anything about `marker` and it will complain about an unknown lint if you run simple `cargo check/build`.
```